lib/dom-event-delegate.js

```diff
@@ -50,6 +60,12 @@ Box.DOMEventDelegate = (function() {
 		// Without checking the existence of a parentNode and returning null, we would throw errors
 		while (!found && element && !isModuleElement(element)) {
 			element = element.parentNode;
+
+			// Stop if we hit a non-basic HTML element such as the DocumentNode
+			if (!isBasicElement(element)) {
+				break;
```

Comment:
Yeah. Anything in the DOM that does not have a `parentNode` isn't an HTMLELement. (It's possible to have a detached element where `parentNode` is `null`.) In this scenario, the first time you'd hit an object without `parentNode` is when `element === document`.